### DISCUSSION TITLE
How to run prism mock under specific localhost route

### DISCUSSION THREAD

**Discussion by 403-html** [stoplightio/prism#2020] [Q&A]
Hello, so I have question, is it possible to run prism mock under specific localhost route?

For example when I pass
```
prism mock -p 8003 swagger.yaml
```
I get all API routes mocked under
```
localhost:8003/{API}
```

but how to achieve that I'll get responses from here (nested to `/api`)?
```
localhost:8003/api/{API}
```

**Comment by crsmly** [comment]
Did you ever find a solution?

**Reply by 403-html** [reply]
Nope, I'm asking because I want to move to this mock software but can't because of that. Just searching alternatives.

**Reply by crsmly** [reply]
muonsoft/openapi-mock seems to be a good alternative.

**Comment by mrashad10** [comment]
Hi
I'm not sure if I do understand what do want, but did you try to add the prefix "/api/" to all your endpoints in the yaml file?
